engine/runtime/src/main/scala/org/enso/compiler/core/IR.scala

```diff
@@ -7395,6 +7395,13 @@ object IR {
           }
 
       }
+
+      case class MissingLibraryImportInFQNError(namespace: String)
+          extends Reason {
+        override def explain(originalName: IR.Name): String =
+          s"Fully qualified name includes a library $namespace.${originalName.name} but import statement is missing."
```

Comment:
Just a very minor suggestion, feel free to ignore.

```suggestion
          s"Fully qualified name references a library $namespace.${originalName.name} but an import statement for it is missing."
```

IMHO `references` reads better, as `includes` suggests that it _does do the import_ (at least to someone biased with C/C++ terminology). Reference seems a bit more neutral and specific about what is happening.